src/select/src/components/styled/selectStyles.js

```diff
@@ -0,0 +1,57 @@
+import { css } from 'styled-components';
+
+export default css`
+  .lattice-select__control {
```

Comment:
Since `lattice-select` is an important prefix for these classnames, perhaps you can use a constant instead?

```
const LATTICE_SELECT = 'lattice-select';
```